dotnet/samples/Hello/HelloAIAgents/HelloAIAgent.cs

```diff
@@ -0,0 +1,34 @@
+using Microsoft.AutoGen.Abstractions;
+using Microsoft.AutoGen.Agents;
+using Microsoft.Extensions.AI;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Hello;
+[TopicSubscription("HelloAgents")]
+public class HelloAIAgent(
```

Comment:
Would it be a good idea to have this inherit from `InferenceAgent` to show an example ? I see that it has an `client`

During runtime this becomes a `LoggingChatclient` how does this happen ?

Reply:
coming up in another pr